pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -539,7 +539,9 @@ func (rc *reconciler) updateStates(volumesNeedUpdate map[v1.UniqueVolumeName]*re
 			continue
 		}
 		if volume.pluginIsAttachable {
-			err = rc.actualStateOfWorld.MarkDeviceAsMounted(volume.volumeName)
+			// devicePath is empty because reconstructedVolume can't obtain devicePath by reading
+			// the pod's volume directories
+			err = rc.actualStateOfWorld.MarkDeviceAsMounted(volume.volumeName, volume.devicePath)
```

Comment:
Please see the comments in previous PR https://github.com/kubernetes/kubernetes/pull/35433#pullrequestreview-6748419

I think it is reasonable to add waitForAttach call here. Before reconstructing the actual state, we should check whether volume is still attached or not. Also we can get the correct devicePath from there.

Reply:
Added WaitForAttach() call.